shared/common-adapters/icon.desktop.js

```diff
@@ -9,15 +9,23 @@ export default class Icon extends Component {
   props: Props;
 
   render (): ReactElement {
-    const color = this.props.style && this.props.style.color || (this.props.opacity ? globalColors.grey1 : globalColors.grey2)
-    const hoverColor = this.props.style && this.props.style.hoverColor || (this.props.opacity ? globalColors.black : globalColors.grey1)
+    let color = null
+    let hoverColor = null
+
+    if (this.props.inheritColor) {
```

Comment:
allow the colors to come from outside this component. we should discuss this pattern as we use it in some complex cases with composite things that have hover interactions